.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
+      actions:
+        patterns:
+          - "*"
+    labels:
+      - "github-actions"
```

Comment:
This label doesn't exist, so I'd drop it.

We could use ["Build / CI"](https://github.com/scikit-learn/scikit-learn/issues?q=is%3Aopen+label%3A%22Build+%2F+CI%22+sort%3Aupdated-desc) though